code/game/objects/items.dm

```diff
@@ -387,7 +388,20 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 // for items that can be placed in multiple slots
 // note this isn't called during the initial dressing of a player
 /obj/item/proc/equipped(mob/user, slot)
-	return
+	if(action_button_name)

```

Comment:
@optimumtact no...
The whole "By default items have one action datum reference, a var to control that one reference's type and a string to decide its name" system.

Its shit.
Action Datums? 100%, Amazing, Greatness.
Action Strings? get_out%, die, you suck